bin/node/runtime/src/lib.rs

```diff
@@ -129,7 +129,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 30300,
```

Comment:
not sure why it shows that the line was updated, same spec used in dev-cere

Reply:
You should increase the `spec_version` before merging to `dev-cere`.

Reply:
please resolve comment, this is outdated

Reply:
You are the author.

Resolved.